helm/values.yaml

```diff
@@ -28,6 +28,28 @@ dbConfig: &dbConfig
     keyFileName: postgresql.key
     caFileName: root.crt
 
+image:
+  repository: openpolicyagent/opa
+  tag: 0.53.0-envoy-3
+
+enabled: true
+replicaCount: 1
+enableLogging: true
+nodePort: 30000
+bundle:
+  serverUrl: http://localhost:9000/opa
+  filePath: /opa.tar.gz
+
+resources:
+  enabled: true
+  value:
+    limits:
+      cpu: 100m
+      memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
```

Comment:
it would be nice to have the values ready for outnet (4 * request = limit)

Reply:
it doesn't really matter as we need to make sizing for opa